phase1-coordinator/src/objects/round.rs

```diff
@@ -1137,82 +1138,16 @@ impl Round {
     /// the [crate::storage::Storage] to reflect the changes to the
     /// round state. `remove_participants` is a list of participants
     /// to remove from the round.
-    pub(crate) fn reset(&mut self, remove_participants: &[Participant]) -> Vec<StorageAction> {
```

Comment:
nice to see lots of code getting deleted :) despite having worked so hard on this!